docs/code-quality/C26400.md

```diff
@@ -35,7 +35,7 @@ else
     buffer = new char[bufferSize];  // C26400
 ```
 
-## Example 2: Simple allocation (fixed with gsl::owner<T>)
+## Example 2: Simple allocation (fixed with gsl::owner\<T>)
```

Comment:
Not sure if this will hit the TOC bug that @mairaw mentioned on the .net repo, because it is an H2